app/src/main/res/values-cat/strings.xml

```diff
@@ -30,53 +30,31 @@
 
     <string name="label_receive_info_gen_qr_code">Premi aquí per mostrar codi QR</string>
 
-    <string name="info_xmrto_enabled">Pagament amb BTC habilitat, premi aquí per més informació.</string>
     <string name="info_ledger_enabled">Ledger habilitat, premi aquí per més informació.</string>
 
-    <string name="info_xmrto"><![CDATA[
-        <b>Heu introduït una adreça %1$s.</b><br/>
-        <i>Està a punt d\'enviar XMR i el destinatari rebrà %2$s a través del servei <b>%3$s</b> service.</i>
-    ]]></string>
-
-    <string name="info_send_xmrto_success_order_label">Ordre %1$s</string>
-
     <string name="info_send_xmrto_success_btc">%1$s %2$s</string>
 
     <string name="info_send_xmrto_paid">Confirmació Pendent</string>
     <string name="info_send_xmrto_unpaid">Pagament Pendent</string>
-    <string name="info_send_xmrto_error">Error de %1$s (%2$s)</string>
     <string name="info_send_xmrto_sent">%1$s Enviats!</string>
     <string name="info_send_xmrto_query">Consultant &#8230;</string>
 
-    <string name="info_send_xmrto_parms"><![CDATA[
-        <b>You can send %1$s &#8212; %2$s %4$s</b>.<br/>
-        <i><b>%5$s</b> està oferint una tasa de canvi de <b>%3$s %4$s/XMR</b> <u>en aquest moment</u></i>.
-    ]]></string>
 
     <string name="send_available_btc">Balanç: %1$s XMR (~%2$s %3$s)</string>
 
     <string name="info_paymentid_integrated">&#x2714; ID de pagment integrat</string>
     <string name="info_prepare_tx">Preparant la seva transacció</string>
 
-    <string name="label_send_progress_xmrto_create">Creant ordre</string>
```

Comment:
why are some used and translated strings removed?

Reply:
I'll have to take a closer look, but what I did was remove any translated strings that have not been modified after the source string was, for safety, basically. This could have gotten rid of a few valid strings.

Reply:
we should leave all translated strings in, removing only english ones